### PR TITLE
Bubble Tea examples

### PR DESCRIPTION
**Pull request by hansvb**

### CODE REVIEW

**Comment by hansvb** [comment]
I chose `rog.rmx-research.tui-bubteasers` as a module name. Haven't given module-names much thought before so you're welcome to change it.
Also it already has golang `1.19` as minimum version. I'm actually not sure what's the best practice here: choose the minimum go version it will work it or just go for the newest and don't care?
Feel free to immediately merge this after approval!

**Review by pmoieni — APPROVED** [review]

**Pull request merged**
